File: go/darts/darts.go
package darts

import "math"

func Score(x, y float64) int {
	x= math.Abs(x)
	y = math.Abs(y)
	res := x+y
	
	if res >= 0 && res < 1.5 {
		return 10
	}else if res >= 1.5 && res <= 7 {
		return 5
	}else if res >= 7.2 && res <= 14{
		return 1
	}
	return 0
}

File: typescript/darts/darts.ts
export function score(x: unknown, y: unknown): unknown {

  
  const xNum = Number(x);
  const yNum = Number(y);

  const distanceSquared = Math.pow(xNum,2) + Math.pow(yNum,2);

  if (distanceSquared > 100) {
    return 0; 
  } else if (distanceSquared > 25) {
    return 1; 
  } else if (distanceSquared > 1) {
    return 5; 
  } else {
    return 10;
  }
}
